lib/galaxy/model/migrate/versions/0027_request_events.py

```diff
@@ -48,10 +48,12 @@ def localtimestamp():
     def nextval(table, col='id'):
         if migrate_engine.name in ['postgres', 'postgresql']:
             return "nextval('%s_%s_seq')" % (table, col)
-        elif migrate_engine.name in ['mysql', 'sqlite']:
+        elif migrate_engine.name in ['mysql']:
             return "null"
+        elif migrate_engine.name in ['sqlite']:
+            return "rowid"
```

Comment:
Revert also this then?

Reply:
Yup - I'll do that when I collapse this down to one commit. Thanks!

We should also probably create a shared library for these migrations so this isn't copied and pasted around.